src/nodes.coffee

```diff
@@ -404,6 +404,13 @@ exports.InfinityLiteral = class InfinityLiteral extends NumberLiteral
   compileNode: ->
     [@makeCode '2e308']
 
+exports.NaNLiteral = class NaNLiteral extends NumberLiteral
+  constructor: ->
+    super 'NaN'
+
+  compileNode: ->
+    [@makeCode '(0/0)']
```

Comment:
If you wanted to do it the better way, you could either create an `Op` node that represents the division and call compile on that, or you could just add the parens yourself by checking `o.level`. But this works fine for a first implementation if you just feel like leaving it. It's up to you.